src/main/java/com/google/cloud/teleport/options/WindowedFilenamePolicyOptions.java

```diff
@@ -0,0 +1,64 @@
+package com.google.cloud.teleport.options;
```

Comment:
This file seems to be the cause of the `import/copybara` failure preventing merging. You'll need to add a license header similar to the other files. You could copy/paste from another file, though please update the year to 2021.

Reply:
Thanks, I'll add that. 

`mvn clean install` seems to complain about a missing _package-info_ file for the new _options_ package, so I'll add that as well.

Reply:
Thanks, I'll add that.

`mvn clean install` seems to complain about a missing _package-info_ file for the new _options_ package, so I'll add that as well.